test/envs/regional-dr-hubless.yaml

```diff
@@ -21,6 +21,7 @@ templates:
     addons:
       - volumesnapshots
       - csi-hostpath-driver
+      - metrics-server
```

Comment:
Is this really needed in this env? this is a very special env for testing ramen without a hub. I would like to keep it simple as possible.

Reply:
Like I said above, I was trying to keep the minikube addons list the same everywhere where we use VMs. I would prefer if we can abstract away the minikube configuration to a different file altogether but that is a much bigger task.

Reply:
Yes it would be nicer if we could remove the duplication in the normal and hubless envs but we can do this later. I would not change this env since it takes lot of time to test it and we don't use it usually.